plugin/online-thesaurus.vim

```diff
@@ -35,6 +35,7 @@ let s:path = shellescape(expand("<sfile>:p:h") . s:script_name)
 
 
 function! s:Lookup(word)
+    let reg_save = @@
```

Comment:
Instead of saving the register here, we can just use the blackhole register on line 50 by replacing it with `1,$d _`